PowerEditor/installer/nativeLang/hongKongCantonese.xml

```diff
@@ -1,12 +1,12 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <NotepadPlus>
-	<Native-Langue name="香港廣東話" filename="hongKongCantonese.xml" version="8.5">
+	<Native-Langue name="香港廣東話" filename="hongKongCantonese.xml" version="8.8.6">
```

Comment:
It should be 8.8.5:
[5. For the PR of translation, don't guess or use the next version number. Use the current version number instead.](https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/CONTRIBUTING.md#guidelines-for-pull-requests)